themes/fresh/layouts/partials/css.html

```diff
@@ -5,6 +5,7 @@
 {{- $fontFace     := replace .Site.Params.font.name " " "+" }}
 {{- $fontSizes    := delimit .Site.Params.font.sizes "," }}
 {{- $fontUrl      := printf "https://fonts.googleapis.com/css?family=%s:%s" $fontFace $fontSizes }}
+<link href="{{ .Site.BaseURL }}posts/index.xml" rel="alternate" type="application/rss+xml" title="{{ .Site.Title }}" />
```

Comment:
On the preview at least, this doesn't contain a trailing slash, resulting in `https://deploy-preview-54--deislabsio.netlify.appposts/index.xml`